### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by KarlW-JCU**
Hello @REJCU

Could you please review my code?
Feedback points:

variable names
following style conventions
readability

Thank you!

### CODE REVIEW

File: prac_06/REFLECTION.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# CP1404 Practical Reflection
```
Comment (REJCU) [inline prac_06/REFLECTION.md:1]: You have the incorrect reflection. 

There is a new one at the bottom of prac_06 before the practice and extension work

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    def __init__(self, name="", year=0, cost=0.0):
+        """Construct guitar object."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return guitar object summary."""
+        return "{} ({}) : ${:,.2f}".format(self.name, self.year, self.cost)
```
Comment (REJCU) [inline prac_06/guitar.py:18]: Down to personal preference but using f{} instead of format for improved readability

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            cost = float(input("Cost: $"))
+            new_guitar = Guitar(name, year, cost)
+            guitars.append(new_guitar)
+            print(f"{new_guitar} added.\n")
+        except ValueError:
+            print("Invalid input: Start again.")
+        name = input("Name: ")
+
+
+def add_test_objects(guitars):
```
Comment (REJCU) [inline prac_06/guitars.py:33]: Personal preference but add test objects is too vague.

File: prac_06/guitar.py (hunk 2 above)
Comment (KarlW-JCU) [reply prac_06/guitar.py:18]: Thanks for the feedback, I too prefer f-strings, but the question contained:
"__str__ - which uses {} string formatting..."
Which I interpreted as using .format, and it is worth practising other methods.

File: prac_06/REFLECTION.md (hunk 1 above)
Comment (KarlW-JCU) [reply prac_06/REFLECTION.md:1]: Thank you for pointing that out.
I copied the first reflection to every second week at the start of the subject.

**Pull request merged**